microsoft-365/compliance/endpoint-dlp-using.md

```diff
@@ -58,24 +58,36 @@ You can use this logic to construct your exclusion paths:
 
 - A mix of all the above. <br/>For example: %SystemDrive%\Users\*\Documents\*(2)\Sub\
 
-### Service domains
+### Unallowed apps
+
+When a policy's **Access by unallowed apps and browsers** setting is turned on and users attempt to use these apps to access a protected file, the activity will be allowed, blocked, or blocked but users can override the restriction. All activity is audited and available to review in activity explorer.
+
+[!IMPORTANT]
+Do not include the path to the executable, but only the executable name (i.e., browser.exe).
+
+
+### Browser and domain restrictions
+Restrict sensitive files that match your policies from being shared with unrestricted cloud service domains.
 
-You can add domains to this list that Edge Chromium will refer to when enforcing the Endpoint DLP policy cloud upload access restriction. 
+#### Service domains
+
+You can control whether sensitive files protected by your policies can be uploaded to specific service domains from Microsoft Edge.
 
 If the list mode is set to **Block**, then user will not be able to upload sensitive items to those domains. When an upload action is blocked because an item matches a DLP policy, DLP will either generate a warning or block the upload of the sensitive item.
```

Comment:
```suggestion
If the list mode is set to **Block**, then the user will not be able to upload sensitive items to those domains. When an upload action is blocked because an item matches a DLP policy, DLP will either generate a warning or block the upload of the sensitive item.
```